beamformer/beamform_reorder/prebeamform_reorder.py

```diff
@@ -50,9 +50,12 @@ def __init__(
         self._sample_bitwidth = 8
         self.complexity = 2
 
-        # n_blocks is hardcoded to 16 and initially started with the tensor core X-Engine kernel and it has shown to allow for efficient thread block size.
-        self.n_blocks = 16
-        self.n_samples_per_block = n_samples_per_channel // self.n_blocks
+        # This 128 is hardcoded in the original tensor core kernel. The reason it is set to this needs to be determined.
+        self.n_samples_per_block = 128 // self._sample_bitwidth
+
+        # n_blocks is hardcoded to 16 and initially started with the tensor core X-Engine kernel and it has shown to
+        # allow for efficient thread block size.
+        self.n_blocks = self.n_samples_per_channel // self.n_samples_per_block

```

Comment:
```suggestion
        # This 128 is hardcoded in the original tensor core correlation kernel to allow for efficient thread-block size
        self.n_samples_per_block = 128 // self._sample_bitwidth
        self.n_blocks = self.n_samples_per_channel // self.n_samples_per_block
```

Just updating the comment to reflect the actual state that the code is in.